<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiPageWidget Example</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QSint::MultiPageWidget" name="MultiPageWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Button Area</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="AreaSelector">
          <item>
           <property name="text">
            <string>Top</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bottom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Alignment</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget1" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="AlignLeft">
             <property name="text">
              <string>Left</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="AlignHCenter">
             <property name="text">
              <string>HCenter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="AlignRight">
             <property name="text">
              <string>Right</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="AlignJustify">
             <property name="text">
              <string>Justify</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="AlignTop">
             <property name="text">
              <string>Top</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="AlignVCenter">
             <property name="text">
              <string>VCenter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="AlignBottom">
             <property name="text">
              <string>Bottom</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>98</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSint::MultiPageWidget</class>
   <extends>QWidget</extends>
   <header location="global">QSint</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
